lua/nvim-tree/marks/init.lua

```diff
@@ -70,7 +70,6 @@ end
 
 function M.setup(opts)
   vim.fn.sign_define(SIGN_NAME, { text = opts.renderer.icons.glyphs.bookmark, texthl = "NvimTreeBookmark" })
-  require("nvim-tree.marks.bulk-move").setup(opts)
```

Comment:
Was that removed accidentally?

Reply:
That setup function was removed as it was no longer needed.